examples/resources/kentik-cloudexport_item/provider.tf

```diff
@@ -2,7 +2,7 @@
 terraform {
   required_providers {
     kentik-cloudexport = {
-      version = "=> 0.2.0"
+      version = ">= 0.2.0"
       source  = "kentik.com/automation/kentik-cloudexport"
```

Comment:
`terraform` will barf on this directive if it is in a file imported via a module. BTW, we should make all of the "source" directives in all example and demo resource files read `source = "kentik/kentik-cloudexport"`, so that the provider is fetched from the registry by default.

Reply:
The examples are intended to be run against locally-built provider, as the header line states. It's useful during development.
I can add a separate collection of examples for public provider. Do we want that?

Reply:
I prefer the examples to run again the public version of the provider. Customers may use the examples as a starting point for development of their own resource files. I don't see any downside in using the public version of the provider. Do you?

Reply:
OK, updating examples to use provider from terraform registry.